src/custom-element.js

```diff
@@ -221,6 +221,9 @@ function createBaseCustomElementClass(win) {
       /** @private {!./size-list.SizeList|null|undefined} */
       this.heightsList_ = undefined;
 
+      /** @public {boolean} */
+      this.has3pOverflow = false;
```

Comment:
How about overriding `getOverflowElement()` in the A4A custom element... (continued below)